dashboard/test/ui/step_definitions/applab.rb

```diff
@@ -182,6 +182,14 @@ def set_nth_input(n, value)
   @browser.execute_script(script)
 end
 
+And /^I drag the instructions grippy by ([-|\d]+) pixels$/ do |delta|
+  drag_grippy('$(".fa-ellipsis-h").eq(0).parent()', 0, delta)
```

Comment:
I considered that. My reason for not doing that is that as soon as it has an id, I (as someone looking at the component's code) need to worry about whether there's some CSS or other logic specific to the id that that lives somewhere else. With no id, I can be reasonable confident it is self-contained.